src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -150,9 +150,9 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
       labels = Map("a" -> "b", "bam" -> "yes"),
       jupyterExtensionUri = jupyterExtensionUri)
 
-    dbFutureValue { _.clusterQuery.save(c1) } shouldEqual c1
-    dbFutureValue { _.clusterQuery.save(c2) } shouldEqual c2
-    dbFutureValue { _.clusterQuery.save(c3) } shouldEqual c3
+    dbFutureValue { _.clusterQuery.save(c1, "gs://bucket") } shouldEqual c1
+    dbFutureValue { _.clusterQuery.save(c2, "gs://bucket") } shouldEqual c2
+    dbFutureValue { _.clusterQuery.save(c3, "gs://bucket") } shouldEqual c3
 
     dbFutureValue { _.clusterQuery.listByLabels(Map.empty, false) }.toSet shouldEqual Set(c1, c2)
     dbFutureValue { _.clusterQuery.listByLabels(Map("bam" -> "yes"), false) }.toSet shouldEqual Set(c1)
```

Comment:
It might be nice to have a case in `LeonardoServiceSpec` as well to verify that the init bucket is saved to the database when creating a cluster.